pkgs/tools/networking/mockoon/default.nix

```diff
@@ -1,15 +1,16 @@
 { lib
 , appimageTools
 , fetchurl
+, nix-update-script
 }:
 
 let
   pname = "mockoon";
-  version = "6.1.0";
+  version = "8.1.1";
 
   src = fetchurl {
-    url = "https://github.com/mockoon/mockoon/releases/download/v${version}/mockoon-${version}.AppImage";
-    hash = "sha256-harZU3TTIzfJoY/jAQI0dm7YSOr24Y9xk9L5ZaBLdD8=";
+    url = "https://github.com/mockoon/mockoon/releases/download/v${version}/mockoon-${version}.x86_64.AppImage";
```

Comment:
please update `meta.platforms` to reflect the reduced platform support

Reply:
I've updated the platforms to an intersection of Linux and x86_64